tests/Runtime-MultiDex/Resources/layout/namefixups.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why do we have this file?  It doesn't appear to be used from anywhere, and it's confusing being alongside `NameFixups.axml`.